documentation/source/QuickstartBasicToolkit.md

```diff
@@ -0,0 +1,245 @@
+# The Basic Skills of SG
+
+Learn the basics of model development with SuperGradients. Researchers and machine learning engineers should start here.
+
+
+<details>
+    <summary>1. Train a Model</summary>
+
+##1. Train a Model
+
+0. Imports:
+
+```python
+from super_gradients.common.object_names import Models
+from super_gradients.training import Trainer, models
+from super_gradients.training.metrics.classification_metrics import Accuracy, Top5
+from super_gradients.training.dataloaders.dataloaders import cifar10_train, cifar10_val
+from super_gradients.training.utils.distributed_training_utils import setup_device
+```
+
+1. Call [setup_device()](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/device.md) according to your available hardware and needs:
+
+```python
+setup_device("cpu")
+```
+
+2. Instantiate a [Trainer]() object #TODO: ADD TRAINER API LINK
+```python
+
+trainer = Trainer(experiment_name="my_cifar_experiment", ckpt_root_dir="/path/to/checkpoints_directory/")
```

Comment:
don't we have to call `deci_trainer.init_trainer()` first?

Reply:
Fixed https://github.com/Deci-AI/super-gradients/commit/d9199dfa4e657910a027f3acfd768c85b6a8fa28.